README.md

```diff
@@ -40,9 +40,53 @@ The Library is splitted across multiple repositories for the different supported
 - The Python version can be found at [maxbachmann/rapidfuzz](https://github.com/maxbachmann/rapidfuzz)
 
 
-## Installation
-As of now it it only possible to use the sources directly by adding them to your project. There will be a version on conan in the future.
+## Compilation
 
+RadidFuzz now support CMake.
+To build it you can do :
+
+    git clone https://github.com/maxbachmann/rapidfuzz-cpp.git rapidfuzz-cpp
+    cd rapidfuzz-cpp
+    mkdir build && cd build
+    cmake ..
+    cmake --build .
+    cmake --build . --target install
+
+RapidFuzz exports its targets to CMake. 
+You can easily integrate it in your CMake project with 3 options.
+
+1. include it as a subdirectory:
+Clone this repo (or make a copy) into your project source tree, lets say in `3rdparty/RapidFuzz`folder.
+Then, in your `CMakeLists.txt` use :
+
+    add_subdirectory(rapidfuzz-cpp)
```

Comment:
I think it would make sence to work with the same subdirectory in all examples.
Right now there is `3rdparty/RapidFuzz`, `3rdparty/rapidfuzz-cpp` and `rapidfuzz-cpp`

Maybe add
```
git submodule add https://github.com/maxbachmann/rapidfuzz-cpp.git 3rdparty/RapidFuzz
```
to the descriptions aswell

Reply:
I revamped the README.md a little bit in 3a4e1350